### PR TITLE
feat/components/marketing-buttons

### PR DESCRIPTION
**Pull request by felixmacaspac**
I implemented a new "underline on hover" button component using pure TailwindCSS.

### CODE REVIEW

File: public/components/marketing-buttons/16.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="relative">
```
Comment (markmead) [inline public/components/marketing-buttons/16.html:1]: Why does it need an extra wrapping element with `relative`? I don't think it's required.

File: public/components/marketing-buttons/16.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class="relative"> 
+  <a class="font-bold text-base text-primary text-black after:content-[''] after:absolute after:-bottom-2 after:h-[2px] after:w-0 after:left-1/2 after:bg-black after:transition-all after:duration-300 after:block hover:after:transition-all hover:after:w-full hover:after:left-0 hover:after:duration-300 hover:after:block" href="/download">
```
Comment (markmead) [inline public/components/marketing-buttons/16.html:2]: `test-base text-black after:content-['']` aren't needed, they are applied by default.

There seem to be a lot of classes on the `before` which feels overkill, here's what I created - https://play.tailwindcss.com/AgRmlrUbo5

**Review by markmead — CHANGES_REQUESTED** [review]
Do you think it's worth adding some variants to this component to showcase off the different ways this hover effect looks?

File: public/components/marketing-buttons/16.html (hunk 1 above)
Comment (felixmacaspac) [reply public/components/marketing-buttons/16.html:1]: you're right, it's not required. 😅

File: public/components/marketing-buttons/16.html (hunk 2 above)
Comment (felixmacaspac) [reply public/components/marketing-buttons/16.html]: Hi, @markmead already refactored the code and add some new variations. :))

**Comment by felixmacaspac** [comment]
> all

Done! 🚀

**Comment by markmead** [comment]
> Swap "Download" to "From <Left|Center|Right>"

Just missing the "From" 😅

**Comment by felixmacaspac** [comment]
lol! missed that one. haha!

**Comment by markmead** [comment]
No problem! It all looks good now, I'll merge! 🚀

Thanks for your contribution @felixmacaspac

**Pull request merged**